learn-more.html

```diff
@@ -4,10 +4,136 @@
     <title>Learn More</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <link rel="stylesheet" href="/styles/styles.css"/>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <script src="https://use.fontawesome.com/ca6c48c124.js"></script>
   </head>
+
+
   <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
+    <section id="header">
+        <h1 id="page_heading"><a href="index.html">Learn More</a></h1>
+        <h3 id="heading_subtext">A page for learning more.</h3>
+        <div id="divider_container">
+          <div id="header_divider" class="divider"></div>
+        </div>
+    </section>
+
+
+    <section id="main">
+        <div id="posts_container">
+  
+          
+            <div id="right_container">

```

Comment:
For this container, it would probably look a bit better to either have the image be as wide as the text, or to shrink the text down to the width of the image and center it. Personally the second way would be more appealing but the inconsistency of the widths is something to pay attention to

Reply:
I will also extend this comment to the continue reading page as well